### PR TITLE
feat(Modals)[OK-201] Create hovercards and modals for deletion buttons

### PR DESCRIPTION
**Pull request by adbista**

### CODE REVIEW

File: client/src/components/board-details/BoardPermissions/BoardPermissions.tsx [hunk 1]
```diff
@@ -86,9 +87,12 @@ export const BoardPermissions = () => {
               <TableCell className="text-foreground">{user.addedAt}</TableCell>
               <TableCell className="text-foreground">{user.invitedBy}</TableCell>
               <TableCell className="flex items-center justify-center">
-                <Button variant="outline" className="hover:text-muted-foreground">
-                  <TrashIcon className="h-4 w-4" />
-                </Button>
+                <DeleteCollaboratorButton
+                  username={user.name}
+                  deleteUser={() => {
+                    /*todo: implement collaborator deletion*/
```
Comment (Strattoss) [inline client/src/components/board-details/BoardPermissions/BoardPermissions.tsx:93]: Can you add a task on Jira for this? It sound like a nice stand-alone task candidate.

File: client/src/components/user-boards/board-table/BoardTable.tsx [hunk 2]
```diff
@@ -103,25 +101,18 @@ const BoardTable: React.FC<BoardTableProps> = ({ activeTab }) => {
...
-                      </HoverCard>
+                    <TableCell
+                      className="flex items-center justify-center space-x-1"
+                      onClick={(e) => {
+                        e.stopPropagation();
+                      }}
+                    >
+                      <BoardDeletionButton
+                        deleteBoard={() => {
+                          /*todo: implement board deletion*/
```
Comment (Strattoss) [inline client/src/components/user-boards/board-table/BoardTable.tsx:112]: Perhaps a Jira task?

**Review by Strattoss — APPROVED** [review]

File: client/src/components/user-boards/board-table/BoardTable.tsx (hunk 2 above)
Comment (adbista) [reply client/src/components/user-boards/board-table/BoardTable.tsx:112]: we have jira task for this

File: client/src/components/board-details/BoardPermissions/BoardPermissions.tsx (hunk 1 above)
Comment (adbista) [reply client/src/components/board-details/BoardPermissions/BoardPermissions.tsx:93]: done

**Pull request merged**